Find a functionally equivalent code:android.content.res.Resources.selectSystemTheme:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * @hide
 */

@@ -8,12 +8,15 @@
     if (targetSdkVersion < Build.VERSION_CODES.ICE_CREAM_SANDWICH) {
         return holo;
     }
+    if (targetSdkVersion < Build.VERSION_CODES.CUR_DEVELOPMENT) {
+        return dark;
+    }
     return deviceDefault;
 }

